Configurations/10-main.conf

```diff
@@ -257,7 +257,7 @@ my %targets = (
         # gcc shared build with Sun C. Given that gcc generates faster
         # code [thanks to inline assembler], I would actually recommend
         # to consider using gcc shared build even with vendor compiler:-)
-        #                        -- <appro@openssl.org>
+        #                        -- <https://github.com/@dot-asm>
```

Comment:
Sorry, I think these should be `https://github.com/dot-asm`

Reply:
fixed